dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_36__Add_created_lastupdated_column_in_customicon.sql

```diff
@@ -0,0 +1,3 @@
+alter table customicon add column if not exists "created" timestamp;
+alter table customicon add column if not exists "lastupdated" timestamp;
+alter table customicon add column if not exists "custom" BOOLEAN;
```

Comment:
This flag shouldn't be in the DB for now.
We are not storing default icons in the DB at the moment.
When we will decide to do so we can add this flag.
Having this flag now and propagate it in the different layers is error-prone and quite confusing